src/opencmiss/zincwidgets/graphicseditorwidget.py

```diff
@@ -49,6 +49,8 @@ def __init__(self, parent=None):
         self._ui.streamlines_track_direction_chooser.setEnumsList(GraphicsStreamlines.TrackDirectionEnumToString, GraphicsStreamlines.TrackDirectionEnumFromString) 
         self._ui.streamlines_colour_data_type_chooser.setEnumsList(GraphicsStreamlines.ColourDataTypeEnumToString, GraphicsStreamlines.ColourDataTypeEnumFromString)
         self._ui.line_shape_chooser.setEnumsList(Graphicslineattributes.ShapeTypeEnumToString, Graphicslineattributes.ShapeTypeEnumFromString)
+        DomainTypeEnum = [Field.DOMAIN_TYPE_POINT,Field.DOMAIN_TYPE_NODES,Field.DOMAIN_TYPE_DATAPOINTS,Field.DOMAIN_TYPE_MESH1D,Field.DOMAIN_TYPE_MESH2D,Field.DOMAIN_TYPE_MESH3D,Field.DOMAIN_TYPE_MESH_HIGHEST_DIMENSION]
```

Comment:
This should be `DOMAIN_TYPE_ENUM` and it should be declared at the top of the file.

Reply:
and spaces after comma.